### PR TITLE
Clearer comments

### PR DESCRIPTION
**Pull request by ddd-mtl**
I'm assuming Nico will rename `state` struct and fields to `store`

### CODE REVIEW

File: agent/src/lib.rs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 //! holochain_agent provides a library for managing holochain agent info, including identities, keys etc..
 
+/// Object holding a Agent's identity.
```
Comment (lucksus) [inline agent/src/lib.rs]: "aN Agent's identity"

File: core/src/state.rs [hunk 2]
```diff
@@ -10,6 +10,8 @@ use std::{
     sync::{mpsc::Sender, Arc},
 };
 
+/// The Store of the Holochain instance Object, according to Redux pattern.
+/// Its composed of a Nucleus and Agent state slice.
```
Comment (lucksus) [inline core/src/state.rs]: I'd rather say: "It's composed of all sub-module's state slices. To plug in a new module its state slice needs to be added here."

File: core/src/lib.rs [hunk 3]
```diff
@@ -1,3 +1,5 @@
+//! The library implementing the holochain pattern fo validation rules + local source chain + DHT
```
Comment (zippy) [inline core/src/lib.rs]: s/fo/of/

File: agent/src/lib.rs (hunk 1 above)
Comment (ddd-mtl) [reply agent/src/lib.rs]: fixed.

File: core/src/state.rs (hunk 2 above)
Comment (ddd-mtl) [reply core/src/state.rs]: Fixed. Please approve PR :)

File: core/src/lib.rs (hunk 3 above)
Comment (ddd-mtl) [reply core/src/lib.rs]: fixed

**Comment by Connoropolous** [comment]
@ddd-mtl if you've made the requested fixes, and got approved on the other one you can just use 'dismiss review' and comment 'fixed'. Sorry if you already knew that

**Comment by ddd-mtl** [comment]
@Connoropolous I did not know that, thanks. I never used the 'dismiss review' button, it seemed a bit... you know... dismissive :)

**Comment by lucksus** [comment]
lol ;) I did approve myself. But the diff is much much smaller.. ?!
did parts of this get merged into develop already?

**Comment by ddd-mtl** [comment]
Oops. I might have messed things up a bit with the other PR on ReturnCodes.
Lesson learned: Will not use same directory for doing 2 different PRs.

**Pull request merged**